data_model/src/query/mod.rs

```diff
@@ -1025,15 +1025,15 @@ pub mod trigger {
     }
 
     impl Query for FindTriggerById {
```

Comment:
```suggestion
    impl Query for FindTriggerByHash {
```

and also change it's definition to:
```rs
        pub struct FindTriggerByHash {
            /// The Identification of the trigger to be found.
            pub id: EvaluatesTo<HashOf<Trigger<FilterBox>>>,
        }
```

Reply:
I believe we can also change `FindAllActiveTriggerIds` into `FindAllActiveTriggerHashes`